.eslintrc.js

```diff
@@ -1,280 +1,214 @@
 module.exports = {
-    'env': {
-        'browser': true,
-        'commonjs': true,
-        'es6': true
+    env: {
+        browser: true,
```

Comment:
I use double quotes always in JSON object files, since quotes preferred over double quotes within source code around strings (unless the string has a lot of escaped single quotes), and literals for keys, unless they have unsupported characters.  

Everyone has a personal taste ;-)

I find single quotes make the file lighter than double quotes (less black ink, heheh)